lib/system/generic/xlnx_common/zynqmp_aarch64/sys.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2022, Xilinx Inc. and Contributors. All rights reserved.
+ * Copyright (c) 2022-2023 Advanced Micro Devices, Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/*
+ * @file	generic/xlnx_common/zynqmp_aarch64/sys.c
+ * @brief	machine specific system primitives implementation.
+ */
+
+#include <metal/compiler.h>
+#include <metal/io.h>
+#include <metal/sys.h>
+#include <stdint.h>
+#include "xil_cache.h"
+#include "xil_exception.h"
+#include "xil_mmu.h"
+#include "xscugic.h"
+
+#ifdef VERSAL_NET
+#include "xcpu_cortexa78.h"
+#elif defined(versal)
+#include "xcpu_cortexa72.h"
+#else
+#include "xreg_cortexa53.h"
+#endif /* defined(versal) */
+
+
```

Comment:
```
 ./scripts/checkpatch.pl --strict --codespell 237.patch 
237.patch:87: CHECK:LINE_SPACING: Please don't use multiple blank lines
#87: FILE: lib/system/generic/xlnx_common/zynqmp_aarch64/sys.c:28:
+
+

```

Reply:
Sorry! thanks @arnopo  i ran checkpatch and the only remaining lines are due to Xilinx-BSP functions now (same as it was). latest commits reflect the fix. thanks